pkg/services/sqlstore/org.go

```diff
@@ -149,8 +149,8 @@ func (ss *SQLStore) CreateOrgWithMember(name string, userID int64) (models.Org,
 	return createOrg(name, userID, ss.engine)
 }
 
-func CreateOrg(ctx context.Context, cmd *models.CreateOrgCommand) error {
-	org, err := createOrg(cmd.Name, cmd.UserId, x)
+func (ss *SQLStore) CreateOrg(ctx context.Context, cmd *models.CreateOrgCommand) error {
+	org, err := createOrg(cmd.Name, cmd.UserId, ss.engine)
```

Comment:
Since you're changing from a function to a method maybe make sense to fix the following at the same time? Context is not forwarded to `createOrg`.

Reply:
Will look into and make a separate PR for both comments 🙂  - thank you!